Packages/io.chainsafe.web3-unity/Editor/ServerSettings.cs

```diff
@@ -196,6 +198,7 @@ private void OnGUI()
         chainID = EditorGUILayout.TextField("Chain ID: ", chainID);
         symbol = EditorGUILayout.TextField("Symbol: ", symbol);
         blockExplorerUrl = EditorGUILayout.TextField("Block Explorer: ", blockExplorerUrl);
+        analyticsOptOut = EditorGUILayout.Toggle("Analytics Opt Out: ", analyticsOptOut);
```

Comment:
Change the toggle text to "Consent to Data Collection for Analytics" and have it on by default

Reply:
Agree! Or also have a tooltip when you hover over it that explains it a bit more.

Reply:
much better